aptos-move/framework/aptos-framework/sources/primary_fungible_store.move

```diff
@@ -104,6 +104,24 @@ module aptos_framework::primary_fungible_store {
         fungible_asset::store_exists(primary_store_address(account, metadata))
     }
 
+    
+    /// Get the address of the primary store for the given account.
+    public inline fun primary_store_address_inlined<T: key>(owner: address, metadata: Object<T>): address {
+        let metadata_addr = object::object_address(&metadata);
+        object::create_user_derived_object_address(owner, metadata_addr)
+    }
+
+    /// Get the primary store object for the given account.
```

Comment:
Can you add comments to explain when these should be used over the non-inline versions?

Also separate question - what happens if you change the existing functions to be inline?

Reply:
@runtian-zhou

Reply:
I believe it won't work because inline function won't be compiled into on chain module. cc @wrwg